doc/2/controllers/models/update-asset/index.md

```diff
@@ -48,7 +48,12 @@ Method: PUT
                 description: string;
               };
             };
-            readOnly?: boolean;
+            definition?: {
+              isReadOnly?: boolean;
+              type?: string;
+              values?: string[] | number[] | boolean [];
+              isAllowedToCustom?: boolean;
```

Comment:
I would rather call it customValueAllowed: boolean

Reply:
Same answer as above